data/battle_anim_scripts.s

```diff
@@ -692,6 +692,19 @@ gBattleAnims_Moves::
 	.4byte Move_MIND_BLOWN
 	.4byte Move_PLASMA_FISTS
 	.4byte Move_PHOTON_GEYSER
+    .4byte Move_ZIPPY_ZAP
+    .4byte Move_SPLISHY_SPLASH
+    .4byte Move_FLOATY_FALL
+    .4byte Move_PIKA_POW
+    .4byte Move_BOUNCY_BUBBLE
+    .4byte Move_BUZZY_BUZZ
+    .4byte Move_SIZZLY_SLIDE
+    .4byte Move_GLITZY_GLOW
+    .4byte Move_BADDY_BAD
+    .4byte Move_SAPPY_SEED
+    .4byte Move_FREEZY_FROST
+    .4byte Move_SPARKLY_SWIRL
+    .4byte Move_VEEVEE_VOLLEY
```

Comment:
Small nitpick, but you're using the wrong indentation style for these new moves. Should be tabs, not spaces.